pwndbg/elf.py

```diff
@@ -126,7 +126,6 @@ def reset_ehdr_type_loaded():
     ehdr_type_loaded = 0
 
 
-@pwndbg.abi.LinuxOnly()
```

Comment:
I know `find_elf_magic` is no used anymore for bare metal, but maybe we should leave `LinuxOnly` just to be super sure.
@zachriggle @anthraxx any thoughts?